src/drivers/hermes/hermes.c

```diff
@@ -25,6 +25,12 @@
 #include <linux/module.h>
 #include <linux/pci.h>
 
+#ifdef __HERMES_DEBUG__
+#define hermes_dbg  pr_err
+#else
+#define hermes_dbg(...)
+#endif
```

Comment:
Don't do this. There's pr_debug() for a reason. dynamic debug is much more useful than a specific ifdef.